branding_guidelines.md

```diff
@@ -0,0 +1,90 @@
+</svg> Download
+</a>
+      </a>
+    </td> <!-- Plain version -->
+    <td align="center" valign="middle">
+      <a href="https://github.com/devicons/devicon/blob/master/icons/devicon/devicon-plain.svg" target="_blank">
+        <img width="150" src="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-plain.svg">
+
+**Plain**
+
+---
+<a href="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-plain.svg">
+<svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="octicon octicon-download mr-1">
+    <path fill-rule="evenodd" d="M7.47 10.78a.75.75 0 001.06 0l3.75-3.75a.75.75 0 00-1.06-1.06L8.75 8.44V1.75a.75.75 0 00-1.5 0v6.69L4.78 5.97a.75.75 0 00-1.06 1.06l3.75 3.75zM3.75 13a.75.75 0 000 1.5h8.5a.75.75 0 000-1.5h-8.5z"></path>
+</svg> Download
+</a>
+    </a>
+  </td> <!-- Original wordmark version -->
+    <td align="center" valign="middle">
+      <a href="https://github.com/devicons/devicon/blob/master/icons/devicon/devicon-original-wordmark.svg" target="_blank">
+        <img width="150" src="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-original-wordmark.svg">
+
+**Original Wordmark**
+
+---
+<a href="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-original-wordmark.svg">
+<svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="octicon octicon-download mr-1">
+    <path fill-rule="evenodd" d="M7.47 10.78a.75.75 0 001.06 0l3.75-3.75a.75.75 0 00-1.06-1.06L8.75 8.44V1.75a.75.75 0 00-1.5 0v6.69L4.78 5.97a.75.75 0 00-1.06 1.06l3.75 3.75zM3.75 13a.75.75 0 000 1.5h8.5a.75.75 0 000-1.5h-8.5z"></path>
+</svg> Download
+</a>
+    </a>
+  </td> <!-- Plain wordmark version -->
+    <td align="center" valign="middle">
+      <a href="https://github.com/devicons/devicon/blob/master/icons/devicon/devicon-plain-wordmark.svg" target="_blank">
+        <img width="150" src="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-plain-wordmark.svg">
+
+**Plain Wordmark**
+
+---
+<a href="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-plain-wordmark.svg">
+<svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="octicon octicon-download mr-1">
+    <path fill-rule="evenodd" d="M7.47 10.78a.75.75 0 001.06 0l3.75-3.75a.75.75 0 00-1.06-1.06L8.75 8.44V1.75a.75.75 0 00-1.5 0v6.69L4.78 5.97a.75.75 0 00-1.06 1.06l3.75 3.75zM3.75 13a.75.75 0 000 1.5h8.5a.75.75 0 000-1.5h-8.5z"></path>
+</svg> Download
+</a>
+      </a>
+    </td>
+  </tr>
+</table>
+
+## 📝 License
+
+All of our branding assets are licensed under the [CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0/) license.\
```

Comment:
I notice that here, we are Creative Commons license. However, in `LICENSE.md`, we are using the MIT one.

For me, I'm cool with people modifying some parts of the icon (like changing its color to match their site) but not changing it completely. Perhaps we should just leave it under CC license and clarify the rest in the `branching_guidelines.md`?

Reply:
> I notice that here, we are Creative Commons license. However, in `LICENSE.md`, we are using the MIT one.
> 
> For me, I'm cool with people modifying some parts of the icon (like changing its color to match their site) but not changing it completely. Perhaps we should just leave it under CC license and clarify the rest in the `branching_guidelines.md`?

I specified that we're using MIT for code and everything else (except that the logos of each trademark respectively have their license, so this won't apply), but CC BY-ND 4.0 for *our* icons, our logo. If I understand well, you want to convert the license to be totally CC BY-ND 4.0, which is something I don't think it's a good idea. 😅

Reply:
Sorry for the late reply. If I understand correctly:
- The icons from the other companies in this repo are listed under MIT license AND still have to adhere to the company's logo usage. This means they can change at most the colors of the font icon but not the SVGs themselves.
- Our logo is under CC, which allows people to modify it. However, the modification is limited by our `branding_guidelines.md`

Am I understand this correctly? I'm sorry if I misunderstand some stuff; licenses are not my strongest suits 😅

Reply:
1. Good point, maybe we should mention that all logos fall under their respective licenses and terms. If no license/terms are found, they are licensed under CC BY-ND 4.0.
2. The current license shouldn't allow modifying our logos (it is BY-ND 4.0 now). So we have two options:
- Remove our customized restrictions (unless they just clarify what you can do and can't do with our current license OR if it has the same restrictions/permissions).
- Remove the license and keep our customized guidelines/restrictions.

@devicons/supporter, @devicons/maintainer, what do you think we should do? I'm not sure if we can keep both options. 😅